cirq-core/cirq/ops/gate_operation_test.py

```diff
@@ -16,6 +16,7 @@
 import pytest
 import sympy
 import cirq
+from cirq.ops.gate_features_test import ThreeQubitTestGate
```

Comment:
I'm not familiar with this convention we have of importing variables from other test files in Cirq. Do we do this elsewhere?

Reply:
I think I'll move it to cirq.testing. The current iteration of this PR has them all defined inline, but it's rather cumbersome (and will be much more so once we get to TwoQubitGate and SingleQubitGate). This becomes more cumbersome because while _num_qubits_ has to be defined, it is frequently never called, so you have to run both the test and coverage check on each instance to determine whether the function should `return 3` or `pass`, with slight variations of the test function affecting that.